### ISSUE TITLE
[fast-switch] Switch implements aria-required which is not allowed

### ISSUE THREAD

**Issue by chrisdholt** [microsoft/fast#3458]
The switch component implements `aria-required` which is not allowed per https://www.w3.org/TR/wai-aria-1.1/#switch. The aria-required attribute should be removed from the foundational component.
